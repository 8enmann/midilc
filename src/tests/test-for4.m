main() {
  Sequence b;
  Sequence c;
  Chord d;
  Chord e;
  Chord f;
  Note rest;
  Note g;
  Number i;

  b = new_sequence();
  c = new_sequence();
  g = Fh;

  d = new_chord(Fw,A,C);
  e = new_chord(Gw,B,D);
  f = new_chord(Cw,E,G);


  for(i = 0; i <= 12; i=i+1){
     if(i == 0 || i == 2 || i == 4 || i==5 || i==7 || i==9 || i==11 || i==12){
         c = c + (g .+ i);
    } else {
         if(i%2 == 0){
          c = c + f;
         }
         else {
          c = c + e;
         }
    }
  } 
  c = c + d;
  rest = Rh;
  play(b);
  play(c);
}
