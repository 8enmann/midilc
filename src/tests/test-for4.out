
0,8,65
8,16,67
8,16,71
8,16,62
24,8,67
32,16,67
32,16,71
32,16,62
48,8,69
56,8,70
64,16,60
64,16,64
64,16,67
80,8,72
88,16,60
88,16,64
88,16,67
104,8,74
112,16,60
112,16,64
112,16,67
128,8,76
136,8,77
144,16,65
144,16,69
144,16,60

